src/skin/legacyskinparser.cpp

```diff
@@ -488,6 +489,10 @@ QList<QWidget*> LegacySkinParser::parseNode(QDomElement node) {
         m_pContext->updateVariable(node);
     } else if (nodeName == "Template") {
         result = parseTemplate(node);
+    } else if (nodeName == "DefineSingleton") {
+        parseSingletonDefinition(node);
+    } else if (nodeName == "Singleton") {
+        result = wrapWidget(parseSingletonContainer(node));
```

Comment:
your comment is hard to read because github is eating the brackets.

RJ didn't want to use the template solution because it is less flexible and doesn't allow for explicit widgetgroup sizing definitions.

I could do "SingletonDefinition" and "SingletonContainer" -- but I actually like that we don't use a verb anywhere else because the singleton is a very different type of skin item.